docs/source/FAQ/data_manager_migration.rst

```diff
@@ -0,0 +1,145 @@
+typical  `project.<Collection> endpoints`\* . New collections, at both the Team and Dataset level,
+will be available in v3.4 of Citrine Python.
+
+\*Newly-registered Datasets can be accessible via Project-based methods if pulled into a project
+with `project.pull_in_resource(resource=dataset)`. However, this is not recommended as endpoints
+listing data by projects and the “pull_in” endpoint for datasets will be removed in 4.0.
+
+How does this change my code?
+=============================
+
+The change in behavior is most localized to two sets of operations on Datasets and their
+constituent GEMD data objects: Sharing and Project-based Collections.
+
+Sharing
+-------
+
+**Within a Team**
+
+Previously, sharing a Dataset from one Project to another was a 2-step process: first publishing
+the Dataset to a Team, then pulling the Dataset into the new project. Now that all Datasets are
+assets of teams, sharing within a team is unnecessary. All of the `publish`, `un_publish`, and
+`pull_in_resource` endpoints, when applied to Datasets will undergo deprecation. To be precise, the
+following calls will return a deprecation warning version for Citrine Python versions 3.4 and
+above, and be removed in version 4.0:
+
+.. code-block:: python
+
+    # Publishing a Dataset to a Team will do nothing once Data Manager is activated
+    project.publish(resource=dataset)
+
+    # Un-publishing a Dataset will similarly be a no-op with Data Manager activated
+    project.un_publish(resource=dataset)
+
+    # Pulling a Team into a Project can still be done with Data Manager activated, but not
+    # recommended and will be deprecated
+    project.pull_in_resource(resource=dataset)
+
+**Between Teams**
+
+Sharing a Dataset from one project to another where those projects are in different Teams was a
+3-step process: publishing to the Team, sharing from one Team to another, then pulling into a
+Project. With Data Manager, only the sharing action is needed.
+
+Previous code for sharing My Dataset from Project A in Team A to eventually use in a Training Set
+in Project B in Team B:
+
+.. code-block:: python
+
+    project_a.publish(resource=my_dataset)
+    team_a.share(
+        resource=my_datset,
+        target_team_id=team_b.uid,
+    )
+    project_b.pull_in_resource(resource=my_dataset)
+
+Is now:
+
+.. code-block:: python
+
+    team_a.share(
+        resource=my_datset,
+        target_team_id=team_b.uid,
+    )
+
+Project-based Collections
+-------------------------
+
+As Datasets are now assets of Teams, typical ways to `list()`, `get()`, or otherwise manipulate
+Datasets or data objects within a Project will undergo a deprecation cycle. As of v3.4, these
+endpoints will still work as usual with a deprecation warning, but will be removed in v4.0. It is
+therefore recommended to migrate your code from all project-based listing endpoints as soon as
+possible to adhere to supported patterns and avoid any costly errors.
```

Comment:
```suggestion
As Datasets are now assets of Teams, typical ways to `list()`, `get()`, or otherwise manipulate Datasets or data objects within a Project will undergo a deprecation cycle.
As of v3.4, these endpoints will still work as usual with a deprecation warning, but will be removed in v4.0.
It is therefore recommended to migrate your code from all project-based listing endpoints as soon as possible
to adhere to supported patterns and avoid any costly errors.
```